src/main/fc/fc_tasks.c

```diff
@@ -366,13 +388,19 @@ cfTask_t cfTasks[TASK_COUNT] = {
         .staticPriority = TASK_PRIORITY_HIGH,
     },
 
-    [TASK_BATTERY] = {
-        .taskName = "BATTERY",
-        .taskFunc = taskUpdateBattery,
-        .desiredPeriod = TASK_PERIOD_HZ(50),        // 50 Hz
+    [TASK_BATTERY_ALERTS] = {
+        .taskName = "BATTERY_ALERTS",
+        .taskFunc = taskBatteryAlerts,
+        .desiredPeriod = TASK_PERIOD_HZ(5),        // 5 Hz
         .staticPriority = TASK_PRIORITY_MEDIUM,
     },
 
+    [TASK_BATTERY_SENSORS] = {
```

Comment:
similarly, i didn't really want scheduler dependencies in battery.c.  

check the commit and see what you think - see 210a6fb54ddbd43dc60fd86206c1d155e8027dd3

yes, the code size is smaller now too
```
Linking SPRACINGF3NEO
arm-none-eabi-size ./obj/main/cleanflight_SPRACINGF3NEO.elf
   text	   data	    bss	    dec	    hex	filename
 249532	   6464	  34288	 290284	  46dec	./obj/main/cleanflight_SPRACINGF3NEO.elf
arm-none-eabi-objcopy -O ihex --set-start 0x8000000 obj/main/cleanflight_SPRACINGF3NEO.elf obj/cleanflight_2.0.0_SPRACINGF3NEO.hex
```